doozer/tests/test_reconcile_jira_issues.py

```
@@ -258,6 +267,81 @@ def test_version_below_4_16_skips_entirely(self, mock_connect):
         runtime.build_jira_client.assert_not_called()
         mock_jira_client.create_issue.assert_not_called()
 
+    @patch('doozerlib.cli.images_streams.connect_issue_with_pr')
+    def test_tier_a_skips_wrong_project_key_in_title(self, mock_connect):
+        """
+        When the PR title contains an issue key from a different project,
+        Tier A should skip it and fall through to Tier B.
+        """
+        runtime, pr_map, mock_issue, mock_jira_client = _build_mocks()
```

Ruff (0.15.15)

[warning] Unpacked variable `mock_issue` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)